src/main/kotlin/uk/gov/justice/hmpps/offenderevents/config/ThymeleafConfig.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.hmpps.offenderevents.config
+
+import nz.net.ultraq.thymeleaf.layoutdialect.LayoutDialect
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class ThymeleafConfig {
```

Comment:
have to manually import the configuration bean since the thymeleaf layout has moved packages